src/jspm-hmr-server.ts

```diff
@@ -66,7 +66,7 @@ export function createServer(options: ServerOptions): JspmHmrServer {
 
     const proxy = httpProxy.createProxyServer();
     app.use(proxyRoute, (req, res) => {
-      req.url = req.baseUrl;
+      req.url = `${req.baseUrl}${req.url}`;
```

Comment:
In my case `req.baseUrl` contains `/api/some-service/some-entity/find` and `req.url` contains `/?parameter=test`. Concatenating them seems to yield the correct address. I've tested it with urls without any parameters as well.

If I remove the line the url only consists `/?parameter=test` and fails!